.vscode/settings.json

```diff
@@ -12,7 +12,7 @@
     "python.formatting.provider": "black",
     "python.jediEnabled": false,
     "restructuredtext.confPath": "${workspaceFolder}/docs",
-    "python.pythonPath": "/scratch/xfz39520/pipenv/pmac_motorhome-TUSz-uxP/bin/python",
```

Comment:
I'm pretty sure vscode does not need this setting anymore. It is best removed as it is specific to a user.